--- setpwc-1.3/Makefile.orig	2011-05-21 21:45:31.000000000 +0200
+++ setpwc-1.3/Makefile	2011-05-28 07:07:39.830385056 +0200
@@ -13,7 +13,7 @@
 # files in the program, then also delete it here.
 
 VERSION=1.3
-CFLAGS+=-Wall -O2 -DVERSION=\"$(VERSION)\"
+CFLAGS=-Wall $(OPTCFLAGS) -DVERSION=\"$(VERSION)\"
 LDFLAGS=
 
 OBJS=setpwc.o
@@ -21,7 +21,7 @@
 all: setpwc
 
 setpwc: $(OBJS)
-	$(CC) -Wall -W $(OBJS) $(LDFLAGS) -o setpwc
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS) -o setpwc
 	#
 	# Oh, blatant plug: http://keetweej.vanheusden.com/wishlist.html
 
